blog.css

```diff
@@ -1,254 +0,0 @@
-/* general body CSS below------------------------------------ */
-
-body {
-    background-color: #FFFAF7;
-}
-
-.contain-blog {
-    display: grid;
-    width: 100%;
-    height: 100%;
-    grid-template-columns: 20% 60% 20%;
-    grid-template-rows: 15% 25% 45% 15%;
-    grid-template-areas: " . header . "
-                        " . blog-head . "
-                         " . main . "
-                         " footer footer footer ";
-    grid-gap: 2rem;                
-}
-
-hr {
-    color: #9A8779;
-}
-
-#header {
-    grid-area: header;
-    align-items: center;
-    text-align: center;
-}
-
-#home-top img {
-    text-align: center;
-    width: 40%;
-    height: auto;
-}
-
-#blog-head {
-    grid-area: blog-head;   
-}
-
-#blog-content {
-    grid-area: main;
-}
-
-#footer {
-    grid-area: footer;
-}
```

Comment:
You could create a general css file to hold the code for elements that are repeated across pages, then link to both the general css, and the page specific css which would allow you avoid having to type this section (and any other sections that are repeated i.e. your footer) out multiple times